src/main/java/com/conveyal/taui/controllers/WorkerController.java

```diff
@@ -185,18 +192,83 @@ private String jsonResponse (Response response, int statusCode, Object object) {
         }
     }
 
+    private static class ExtendedJobStatus extends JobStatus {
+        public RegionalAnalysis regionalAnalysis;
```

Comment:
This wrapper subclass class and its copying constructor seem to exist only to add an optional RegionalAnalysis field. It would perhaps be simpler to include this field in the superclass and leave it `null` when it's not needed. But in fact I think this is the only place the class is ever used, in which case the RegionalAnalysis field is always filled in, so we might as well just add the field to the superclass.

Reply:
Ah, for some reason I thought `JobStatus` was in R5 and I was just extending it to keep the changes in the backend.

Reply:
It used to be when the whole broker was in R5, but we have merged the broker into the backend since it's no longer a separate process.